cts/cts-cli.in

```diff
@@ -932,7 +932,19 @@ test_rules() {
     export CIB_shadow=$shadow
 
     cibadmin -C -o resources   --xml-text '<primitive class="ocf" id="dummy" provider="heartbeat" type="Dummy" />'
-    cibadmin -C -o constraints --xml-text '<rsc_location id="no-date-expression" rsc="dummy" score="-INFINITY" node="node01" />'
+
+    TMPXML=$(mktemp ${TMPDIR:-/tmp}/cts-cli.tools.xml.XXXXXXXXXX)
+    cat <<EOF > "$TMPXML"
+<rsc_location id="cli-too-many-date-expressions" rsc="dummy">
+  <rule id="cli-rule-too-many-date-expressions" score="INFINITY" boolean-op="or">
+    <date_expression id="cli-date-expression-1" operation="gt" start="2020-01-01 01:00:00 -0500"/>
+    <date_expression id="cli-date-expression-2" operation="lt" end="2019-01-01 01:00:00 -0500"/>
+  </rule>
+</rsc_location>
+EOF
+
+    cibadmin -C -o constraints -x "$TMPXML"
+    rm -f "$TMPXML"
```

Comment:
Definitely not needed for this PR, but just FYI, cibadmin also supports -p to get the XML from stdin so we could do echo '...' | cibadmin -p ...

Reply:
Yeah, good to know.  I was just trying to fit in with everything else here.